conventions.md

```diff
@@ -188,23 +281,99 @@ Follow the folder structure to locate resources and generate or modify code in a
 
 - **Audit log file (`auditLog.json`):**  
   Contains two main sections:
-  1. **audits:** Each audit entry has a unique ID (e.g., `auditYYYYMMDD`), an auditCompletedOn date, auditedBy, auditorGitHandle (if applicable), auditReportPath (PDF location in `audit/reports/`), and an auditCommitHash.
+  1. **audits:** Each audit entry has a unique ID (e.g., `auditYYYYMMDD_X`) where `YYYYMMDD` = Audit completion date, `_X` = Incrementing counter for multiple audits on the same day. Ensure that contract versioning is consistent between the code and the `auditLog.json`.
+  - Essential fields for each audit entry:
+    - `auditCompletedOn`: Date in `DD.MM.YYYY` or `YYYY-MM-DD` format.  
+    - `auditedBy`: Name or firm.  
+    - `auditorGitHandle`: (if applicable).  
+    - `auditReportPath`: PDF location in `audit/reports/`.  
+    - `auditCommitHash`: The commit hash that was audited (or “n/a” if not tracked).
   2. **auditedContracts:** Maps contract names and versions to the relevant audit IDs in the audits section.
 - **Storing reports:**  
-  Place PDF reports in the `audit/reports/` directory, using a naming format that includes the date and contract info (e.g., `2025.01.17_LiFiDexAggregator(v1.3.0).pdf`).
+  Place PDF reports in the `audit/reports/` directory. The naming format must follow: `YYYY.MM.DD_ContractName(version).pdf` (e.g., `2025.01.17_LiFiDexAggregator(v1.3.0).pdf`).
```

Comment:
@0xDEnYO moving your comment here - https://github.com/lifinance/contracts/pull/1040#discussion_r1979699859

"_maybe even define the file name in format:

YYYY.MM.DD_ContractName(version).pdf

Also we should specify what is the convention in case of multiple contracts being audited. Feel free to suggest sth_"

I'm not sure about this as having multiple contracts within a single file might require a different naming convention. Perhaps we should use `YYYY.MM.DD_FileName(version).pdf` instead of `ContractName`. If the contracts are separated into individual files they would follow our existing convention: `YYYY.MM.DD_ContractName(version).pdf` and there's no need for a new one.

Reply:
How do you picture to file to file the report for the Cantina competition?
This is a full audit across the whole codebase, so it covers every single contract.
That is what I was mainly referring to. 

I would think that we go for something like 
`YYYY.MM.DD_Cantina_Competition_Full_Codebase_Review.pdf` or so and then have entries in the section below that link each contract to this audit.

Reply:
Updated. I choose I went `YYYY.MM.DD_CustomFileName.pdf` format without underscores for audit for multiple contracts